### PR TITLE
qt 6.4.2

### PR DESCRIPTION
**Pull request by p-linnane**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/qt.rb [hunk 1]
```diff
@@ -136,14 +135,6 @@ class Qt < Formula
...
-  # Fix Linux build with CMake versions >= 3.25
-  # remove in next release
-  patch do
-    url "https://github.com/qt/qtwebengine/commit/240e71877865ed07e4c8d5bd4553aa0772c2adf4.patch?full_index=1"
-    sha256 "8fb13bfc7aac50084e1c533955564a1819bbb25b544ebccd05b99e24527c7b80"
-    directory "qtwebengine"
-  end
-
   def install
     python = "python3.10"
```
Comment (cho-m) [inline Formula/qt.rb:139]: May be able to update to `python@3.11` as I think the backport commit is in this release.
- https://github.com/qt/qtwebengine/tree/v6.4.2/src 
- https://github.com/qt/qtwebengine-chromium/tree/0a34013e271bd8831255bb01c9c8284459d17cca

Should be able to supersede #119090 if it works.

Comment (p-linnane) [reply Formula/qt.rb:139]: Let's give it a shot!

Comment (cho-m) [reply Formula/qt.rb:139]:
```suggestion
    python = "python3.11"
```

**Comment by p-linnane** [comment]
Failure on Monterey Intel was due to network flakiness.

**Review by cho-m — APPROVED** [review]

**Pull request closed without merging**